docs/enterprise-config.md

```diff
@@ -0,0 +1,57 @@
+# Enterprise configuration defaults
+
+Git Credential Manager Core (GCM Core) can be configured using multiple
+different mechanisms. In order of preference, those mechanisms are:
+
+1. [Environment variables](environment.md)
+2. [Standard Git configuration files](configuration.md)
+   1. Repository/local configuration (`.git/config`)
+   2. User/global configuration (`$HOME/.gitconfig` or `%HOME%\.gitconfig`)
+   3. Installation/system configuration (`etc/gitconfig`)
+3. Operating system specific configuration store
+
+This model largely matches what Git itself supports, namely environment
+variables that take precedence over Git configuration files.
+
+The addition of the operating system specific configuration store enables system
+administrators to configure defaults for many settings that may be required in
+an enterprise or corporate setting.
```

Comment:
maybe add something like "[3] may allow administrators to use normal enterprise tooling instead of [2.3] to simplify their configuration tasks."

I guess I'm looking for a motivation for adding [3] given that we already have [2.3].